src/stim/dem/detector_error_model.pybind.h

```diff
@@ -22,9 +22,10 @@
 namespace stim_pybind {
 
 pybind11::class_<stim::DetectorErrorModel> pybind_detector_error_model(pybind11::module &m);
-void pybind_detector_error_model_after_types_all_defined(pybind11::module &m, pybind11::class_<stim::DetectorErrorModel> &c);
-std::string detector_error_model_repr(const stim::DetectorErrorModel &self);
+void pybind_detector_error_model_methods(pybind11::module &m, pybind11::class_<stim::DetectorErrorModel> &c);
 
 }
 
-#endif
\ No newline at end of file
+std::string detector_error_model_repr(const stim::DetectorErrorModel &self);
```

Comment:
Keep this in the namespace. It's a method that exists to facilitate the python API so it goes in the stim_pybind namespace.

Reply:
:+1: